db/migrate/20190717180516_create_join_table_user_kindergarten.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinTableUserKindergarten < ActiveRecord::Migration[5.2]
```

Comment:
can u delete this migration file if you are not using it

Reply:
We have connected users to kindergartens and kindergartens to users in both directions with a has_and_belongs_to_many relationship.  Then, at the end of the feature test, we link a user to at least one kindergarten so I believe it stores the link information in the Join Table.

Reply:
but you have commented out code in this migration file?

Reply:
The migration file has been removed.